.github/workflows/build-docs.yml

```diff
@@ -43,12 +43,12 @@ jobs:
       with:
         path: .venv
         key: venv-${{ runner.os }}-${{ steps.setup-python.outputs.python-version }}-${{ hashFiles('**/poetry.lock') }}
-
+        
```

Comment:
These white spaces seem unnecessary, but not too tragic.

Reply:
Removed the whitespaces